### DISCUSSION TITLE
My New Show -<code><br></code>

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#11] [Comments]
# My New Show -<code><br></code>

Announcing my new show - <Code><Br>

https://www.raymondcamden.com/2024/01/22/my-new-show-codebr

**Comment by bobmonsour** [comment]
No particular idea of what I'd like to hear, but I'm looking forward to tuning in. Ok, now that I've written that, I'd like to know how you organize your file system with a broad array of projects that you work on.

**Reply by cfjedimaster** [reply]
For my "this is my stack" show, I was absolutely considering stuff like this that isn't software per se. I'll add it to my list!

**Comment by daniel2design** [comment]
good idea, wish you good luck!

**Comment by ecarlisle** [comment]
I'd love advice, discussion on evangelizing Composable Architecture. That's a sell I've been trying to make at multiple engagements.
